### DISCUSSION TITLE
[Feature Request]: Need to track "Internal_Temperature" of inverter

### DISCUSSION THREAD

**Discussion by MichaelKreitmann** [wlcrs/huawei_solar#406] [General]
### Describe your feature request

There is already a sensor defined "Internal_Temperature" but it is not visible inside and Dashboard component as available sensor ? 
How to access this ?
Thanks!

### Proper usage

- [X] I confirm that this is not a bug report
- [X] I confirm that this feature request is within the stated scope of the integration

**Comment by selbstmacher** [comment]
I would also like to read out the internal temperature to prevent that the internal fan turns on, because the internal is very annoying.

**Reply by selbstmacher** [reply]
Just found out that you only have to enable the entity.
This is how you do activate the internal Temperature:

1. Go to "Settings"
2. Open "Devices and Services"
3. Click on "Huawei Solar"
4. Click on "Entities"
5. Search for "sensor.inverter_internal_temperature"
6. Open Entity
7. Click on settings symbol on the upper right corner
8. Set Entity to Enabled and Visible
9. Restart Home Assistant

Voila internal Temperature is now available.
Maybe you don`t have to restart HA, but I did it.

**Reply by koehntopp** [reply]
Duh - that was easy... ;)

Thank you - I have just installed feans on top of the heat sink, and I wanted to see their effectiveness.

**Comment by ruteclrp** [comment]
Enable the hidden entity in the device
